source/usage-examples/updateMany.txt

```diff
@@ -15,29 +20,6 @@ Update Many Documents
 You can update more than one document using the ``UpdateMany()`` method on
 a collection object.
 
-Example
--------
-
-The examples on this page use the following ``Restaurant``, ``Address``, and ``GradeEntry`` 
-classes as models:
-
-.. literalinclude:: /includes/code-examples/Restaurant.cs
-   :language: csharp
-   :copyable:
-   :dedent:
-
-.. literalinclude:: /includes/code-examples/Address.cs
-   :language: csharp
-   :copyable:
-   :dedent:
-
-.. literalinclude:: /includes/code-examples/GradeEntry.cs
-   :language: csharp
-   :copyable:
-   :dedent:
-
-.. include:: /includes/convention-pack-note.rst
-
 The following code updates all documents in the ``restaurants`` collection that have a 
```

Comment:
I: this file is still missing the Example heading

Reply:
doh, forgot to save after making that change